api/envoy/api/v2/listener/quic_config.proto

```diff
@@ -30,4 +32,8 @@ message QuicProtocolOptions {
   // Connection timeout in milliseconds before the crypto handshake is finished.
   // 20000ms if not specified.
   google.protobuf.Duration crypto_handshake_timeout = 3;
+
+  // Runtime flag that controls whether the listener is enabled or not. If not specified, defaults
+  // to enabled.
+  core.RuntimeFeatureFlag enabled = 4;
```

Comment:
Just chiming in here late, but curious why we needed the ability to runtime enable UDP on a per-listener basis, i.e. why it's not sufficient to do runtime override proxy-wide. I'm guessing most of the time you want this that something has gone seriously wrong serving UDP and you want everyone to fall back to TCP, and that would be proxy-wide. I can see the per-listener argument in a multi-tenant scenario, but I imagine in that case you would be willing to accept an LDS push.

Otherwise API LGTM.

Reply:
@htuch this was agreed upon as optimal approach based on effort level and current state of UDP listeners codebase.  Please also see long discussion here: https://github.com/envoyproxy/envoy/pull/9679#discussion_r369872379

Reply:
OK, resolving, sorry for the redundant thread.